webapp/src/main/java/org/apache/atlas/web/rest/RelationshipREST.java

```diff
@@ -190,8 +186,7 @@ public void deleteByIds(List<String> guids) throws AtlasBaseException {
             if (AtlasPerfTracer.isPerfTraceEnabled(PERF_LOG)) {
                 perf = AtlasPerfTracer.getPerfTracer(PERF_LOG, "RelationshipREST.deleteById(" + guids.size() + ")");
             }
-
-            relationshipStore.deleteByIds(guids);
+            relationshipStore.deleteByIds(guids, true);
```

Comment:
@suraj5077  I see we pass `true` for all actions to notify.. Why it cant be at single place, while notification, check if notifying is enabled or not..

Reply:
This is a different flow @n5nk . Notification check is enabled only at a single place. For that check sendNotifications method in class AtlasRelationshipStoreV2.